frontend/src/views/Quiz.vue

```diff
@@ -112,19 +167,19 @@ function goToPreviousQuestion() {
         </h1>
         <br />
         <QuizPreview
-            v-if="currentQuestionIndex == null"
-            :questionsCount="quiz.questions.length"
+            v-if="currentQuestion == null"
+            :questionsCount="quizQuestions.length"
             :description="quiz.description"
             :authors="quiz.authors"
             :onStart="startQuiz"
         />
         <question
```

Comment:
This looks like a design issue haha. Sorry I should have said that earlier.

When you look at the structure, the Previous and Next buttons have nothing to do with the question. I mean, whatever the question is, it has an `id`, a `previous_question_id` and a `next_question_id`. So, we should extract this component and do something like

```vue
<QuizPreview ... />
<question
                v-else
                :questionId="currentQuestion"
                :nbQuestions="quizQuestions.length" 
/>
<AnotherComponentToHandlePrevAndNext
                :goToNextQuestion="goToNextQuestion"
                :goToPreviousQuestion="goToPreviousQuestion"
/>
```

Reply:
Absolutely ! I should have seen it before.
Let me fix it later.